api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -195,6 +195,15 @@ message HttpConnectionManager {
   // :ref:`config_http_conn_man_headers_x-forwarded-for` for more information.
   uint32 xff_num_trusted_hops = 19;
 
+  message InternalAddressConfig {
+    // Whether unix socket addresses should be considered internal.
+    bool unix_sockets = 1;
+  }
+
+  // Configures what network addresses are considered internal for stats and header sanitazion
+  // purposes.
```

Comment:
Can you link to existing documentation on internal vs external, so that the user knows what is used if this is unset? Or somehow clarify what the default behavior is. Possibly the correct thing to link to: https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/headers.html?highlight=internal#x-envoy-internal